src/Classes/Canvas.ts

```diff
@@ -124,7 +122,7 @@ export class Canvas {
         const container = this.container;
         const options = container.options;
         const trail = options.particles.move.trail;
-        const fillColor = Utils.hexToRgb(trail.fillColor);
+        const fillColor = typeof trail.fillColor === "string" ? Utils.hexToRgb(trail.fillColor) : Utils.colorToRgb(trail.fillColor);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Exceeds maximum line length of 120](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5216054)